### ISSUE TITLE
Learning rate hyperparameter search

### ISSUE THREAD

**Issue by TenzinCHW** [TenzinCHW/Algorithmic-Game-Theory#6]
As above. Perform hyperparameter search over the same number of epochs (mainly from scores I guess)
